### PR TITLE
Add changelog for v6.1.1

### PR DESCRIPTION
**Pull request by msau42**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-6.1.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
 
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Update deployment tags to v6.1.0 (#766, @RaunakShah)
+
+### Uncategorized
+
+- CVE fixes: CVE-2023-44487,  CVE-2023-3978 (#929, @dannawang0221)
```
Comment (xing-yang) [inline CHANGELOG/CHANGELOG-6.1.md:13]: Can you run release-notes with the "--markdown-links" flag?  This way it will provide a markdown link.

**Comment by msau42** [comment]
6.1 branch is unsupported now
/close

**Pull request closed without merging**